Homework2/Program.cs

```diff
@@ -11,9 +11,71 @@
 
 Console.WriteLine();
 bankAccount4.AddMoney(20M);
-Console.WriteLine(bankAccount4);
-bankAccount4.PullOfMoney(50M);
-bankAccount4.PullOfMoney(30M);
-Console.WriteLine(bankAccount4);
+//Console.WriteLine(bankAccount4);
+//bankAccount4.PullOfMoney(50M);
+//bankAccount4.PullOfMoney(30M);
+//Console.WriteLine(bankAccount4);
+
+
+bool result1 = bankAccount3.Transfer(bankAccount4, 20M);
+bool result2 = bankAccount3.Transfer(bankAccount4, 20M);
+
+
+string str1 = "Hello World!";
+Console.WriteLine("Before: {0}, after {1}", str1, Reverse(str1));
+
+
+string path = @"c:\Users\Михаил\source\repos\OOP_Tutorail\Homework2\";
+string inputFile = "inputFile.txt";
+string outputFile = "outputFile.txt";
+string inputPath = Path.Combine(path, inputFile);
+string outputPath = Path.Combine(path, outputFile);
+if (File.Exists(inputPath))
+{
+    StreamReader stream = new StreamReader(inputPath);
```

Comment:
```suggestion
    using var stream = new StreamReader(inputPath);
```
Всегда(!) при работе с файлами не забывайте про конструкцию `using`!!!

Reply:
Правильно ли я понимаю, что в блок using нужно класть работу с объектами, которые требуется закрыть при окончании работы? и каждый ли объект нужно оборачивать в using или весь блок, например, работы с файлом, обернуть будет нормально

Reply:
> 

Правильно ли я понимаю, что в блок using нужно класть работу с объектами, которые требуется закрыть при окончании работы? и каждый ли объект нужно оборачивать в using или весь блок, например, работы с файлом, обернуть будет нормально